src/resources/postcss/tickets.pcss

```diff
@@ -260,6 +257,15 @@
 		}
 	}
 
+	.tribe-tickets__sale_price-suffix {
```

Comment:
```suggestion
	.tribe-tickets__sale-price-suffix {
```

Reply:
I've been trying to keep consistent with the other classes for the template: https://github.com/moderntribe/event-tickets/pull/1639/files/f29ef0695f1d6c72f31e86b3e53ee97c7be77d44#diff-81fcfb5b523c4b2d6e7a09345af69ab0R40 - Would you recommend changing them all?

Reply:
Yeah I see that. I'd say leave what we've already put out for customers so we don't mess up their customizations, but anything new we put out let's fix them.